orttraining/orttraining/models/runner/training_runner.h

```diff
@@ -186,13 +186,13 @@ class TrainingRunner {
 
  private:
   enum SessionMode: int {ModelUpdateStep, GradientAccumulateStep, EvaluateStep};
-  Status PrepareFeedNamesAndFeeds(const SessionMode mode,
-                                  IDataLoader& training_data_loader,
-                                  DataSet& training_data,
-                                  LearningRateScheduler* lr_scheduler,
-                                  const size_t batch_index,
-                                  std::vector<std::string>& feed_names,
-                                  std::vector<MLValue>& feeds);
+  void PrepareFeedNamesAndFeeds(const SessionMode mode,
```

Comment:
nit: why the inconsistency of return type between PrepareFeedNamesAndFeeds() and PrepareFetchNamesAndFetches()? i see one uses ORT_ENFORCE() and the other uses ORT_RETURN_IF(), would be nice to pick either exceptions or return values for consistency

Reply:
Ok. Data preparation functions all returns `Status`.  `RunWithoutUpdate` and `RunWithUpdate` doesn't return `Status` but use `ORT_ENFORCE` inside them. I originally want  all of them to return `Status` but `RunWithoutUpdate` is async, so it can't directly return a `Status`.